src/python/system/environment.py

```diff
@@ -111,9 +106,7 @@ def _parse_memory_tool_options(options_str):
 def _quote_value_if_needed(value):
   """Quote environment value as needed for certain platforms like Windows."""
   result = value
-
-  bot_platform = platform()
-  if bot_platform == 'WINDOWS':
+  if ' ' in result or ':' in result:
```

Comment:
What about other whitespace (e.g. tabs)?

Reply:
We only run this on paths (symbolizer path) and it shouldnt have anything other than spaces. unsure if adding regex is worth it here.